templates/charms-architecture.html

```diff
@@ -0,0 +1,243 @@
+        <ul>
+          <li><strong>Controller agent: </strong>a process running workers responsible for a controller. This includes, among others, the Juju API server.</li>
+          <li><strong>Machine agent: </strong>Found on machine clouds, this is a process running workers responsible for a machine.</li>
+          <li><strong>Model agent: </strong>Found on machine and Kubernetes clouds, this is a juju process running workers responsible for all the models associated with a given controller.</li>
+          <li><strong>Unit agent: </strong>Found on machine and Kubernetes clouds, this isa process responsible for a unit.</li>
+        </ul>
+      </div>
+    </div>
+  </section>
+  <section class="p-section--shallow">
+    <div class="row">
+      <div class="col-start-large-4">
+        <hr class="p-rule--muted">
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-3 col-start-large-4">
+        <p class="p-heading--5">What are pebbles</p>
+      </div>
+      <div class="col-6 col-start-large-7">
+        <p>Pebble is a lightweight, API-driven process supervisor. In charms it is used to give workload containers something akin to an init system that will allow the charm container to interact with them.</p>
+        <p>Pebble allows to start, stop, restart and update services while taking into account service dependencies. Pebble implements custom "health checks" that can be configured to restart services when they fail, as well as the ability to forward logs to a remote Loki server.</p>
+        <p>Pebble also includes a subsystem called notices, which allows the user to introspect various events that occur in the Pebble server, as well as record custom client events. The server saves notices to disk, so they persist across restarts, and expire after a notice-defined interval.</p>
+      </div>
+    </div>
+  </section>
+</section>
+<section class="p-section u-no-padding--bottom">
+  <section class="p-section--shallow">
+    <div class="row--50-50">
+      <div class="u-fixed-width"><hr class="p-rule"></div>
+      <div class="col">
+        <h1 class="p-heading--2">How do charms work?</h1>
+      </div>
+      <div class="col">
+        <p class="p-heading--5">A simple explanation of how charms interact with Juju and the workload they operate</p>
+        <p>The Juju controller is the brain of every deployment and it is a process that runs in the background and manages the relationships between the clients (e.g. the CLI or Terraform), the cloud substrate and the charm code. The controller also maintains the state of the system, as well as other data structures that can be accessed by the charms.</p>
+      </div>
+    </div>
+  </section>
+  <div class="row">
+    <div class="col-start-large-4">
+      <hr class="p-rule--muted">
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-3 col-start-large-4">
+      <p class="p-heading--5">When a client issues a command the following happens:</p>
```

Comment:
I think the spacing looks weird, is there something we can do to make it more visually appealing? 
![image](https://github.com/canonical/juju.is/assets/100278053/df474ec4-d996-4471-a099-58e23c49b138)

Reply:
Updated this section to use regular `p-list-divided` component, rather than the one that was initially suggested by visual design. 
![image](https://github.com/canonical/juju.is/assets/152963078/ef65a95e-5c5a-4c7a-90a9-1cc9f7107a98)

Reply:
way better, love the new design